lib/hammer_cli_foreman_ansible/command_extensions/inventory.rb

```diff
@@ -0,0 +1,114 @@
+# frozen_string_literal: true
+
+module HammerCLIForemanAnsible
+  module CommandExtensions
+    class Inventory < HammerCLI::CommandExtensions
+      output do |definition|
+        definition.append do
+          field :hostname, _('Name')
+          field :fqdh, _('fqdh')
+          field :hostgroup, _('hostgroup')
+          field :location, _('location')
+          field :organization, _('organization')
+          field :domainname, _('domain')
+          field :foreman_domain_description, _('foreman_domain_description')
```

Comment:
I see there is a lot of such "key" labels, which is not quite good for a person to read. Since it may also be translated, I'd say that they should be more "human" :)